docs/notebooks.rst

```diff
@@ -59,6 +59,12 @@ Download the notebooks into a suitable folder:
    notebooks/Find Asset and Create Attachment
    notebooks/Create Event with Verified Domain
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Access Policy Examples
+
+   notebooks/Create Access Policy -- User Read Only
```

Comment:
I know this comment is not this PR but I think naming these notebooks after actual business cases rather than abstract concepts such as 'Access Policy' would be preferable.

Something like 'Allow another Artist to Read Adeles's Album releases' or suchlike

Reply:
Hello @eccles. . . fair point. Honestly as long as we are consistent and keep the name of the notebooks short (think what I named the notebook is too long) then I'm fine either way.  This means we should probably update the names of the other notebooks to ensure consistency.

Thoughts. . . @JAG-UK. . . @robinbryce . . . @honourfish . . . @waarg. . . @Serhiy1